nodejs/week1/avg.js

```diff
@@ -0,0 +1,19 @@
+const args = process.argv.slice(2); // remove first 2 elements;
+
+if (args.length === 0) {
+  console.log("Please provide at least one number.");
+} else {
+  const numbers = args.map(Number); // Convert arguments to numbers
+
+  if (numbers.some(isNaN)) {
+    console.log("Please provide only valid numbers.");
+  } else {
+    let sum = 0;
+    for (let i = 0; i < numbers.length; i++) {
```

Comment:
You used functional APIs in `map`, `some`. You can try to use `reduce` to avoid this for loop.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce